### PR TITLE
teal.widgets - bs345

### PR DESCRIPTION
**Pull request by Polkas**
linked to teal/#709](https://github.com/insightsengineering/teal/issues/719)

- [x] remove not used functionality to record if the panel item is clicked from the server perspective.
- [x] panel group and panel item working under all bootstrap versions 3,4 and 5 
- [x] remove not needed div in the standard_layout to make it working under all bootstrap versions

Example for panel_(group/item):

```
library(shiny)

ui <- fluidPage(
  theme = bslib::bs_theme(version = "4"), # test 3, 4 and 5
  teal.widgets::panel_item("Item1", "Hey"),
  teal.widgets::panel_group(teal.widgets::panel_item("Item1", "Hey"), teal.widgets::panel_item("Item2", "Hey")),
  teal.widgets::panel_item("Item2", "Hey"),
  teal.widgets::panel_item("Item3", "Hey")
)

server <- function(input, output, session) {
  
}

shinyApp(ui, server)
```

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # teal.widgets 0.1.1.9013
 
+### Breaking changes
+* `panel_group` and `panel_item` are no longer offer to be optionally a shiny input.
```
Comment (nikolas-burkoff) [inline NEWS.md:4]: I think this needs to be a little clearer - it's the css class name rather than the panel group itself?

Comment (Polkas) [inline NEWS.md:4]:
```suggestion
* `panel_group` and `panel_item` functions are no longer offer to be optionally a shiny input.
```

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # teal.widgets 0.1.1.9018
 
+### Breaking changes
+* `panel_group` and `panel_item` functions are no longer offer to be optionally a shiny input.
```
Comment (mhallal1) [inline NEWS.md:4]:
```suggestion
* Updated `panel_group` and `panel_item` functions to no longer be an optional shiny input.
```

**Comment by asbates** [comment]
lgtm

**Review by donyunardi — APPROVED** [review]
LGTM

**Pull request merged**